Source/BusinessMomentsAdaptorImage/dataschema.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+const uri: string = 'mongodb://127.0.0.1:27017/local';
+
+mongoose.connect(uri, (err: any) => {
+    if (err) {
+        console.log(err.message);
+    } else {
+        console.log('Successfully Connected to MongoDB!');
+    }
+});
+
+export const DataSchema = new mongoose.Schema({
+    tenant: { type: String, required: false },
+    publisher: { type: String, required: false },
+    operation: { type: String, required: false },
+    document: { type: String, required: false },
+    timestamp: { type: String, required: false },
+    id: { type: String, required: false },
+    elements: { type: Array, required: false },
+    message: { type: String, required: false },
+});
+
+const rawData = mongoose.model('rawData', DataSchema);
```

Comment:
Here is how you can fix the schema issue,

```
const rawData = mongoose.model('rawData', new mongoose.Schema({ any: {} }, { strict: false }));

```

Reply:
Seems this is not working. I will have more dig into it to make a generic schema.

Reply:
My mistake. It is working!